iceoryx_posh/test/moduletests/test_roudi_process.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test.hpp"
+#define private public
```

Comment:
Please remove the public defines, you do not need them. Generally the tests should only use the public API and not depend on internal variables/functions.

Reply:
In windows it is especially bad since here the compiler cannot find the class anymore since public/protected/private are belonging to the class signature and with this define we are changing it.